org.kde.kdiff3.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "org.kde.kdiff3",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kdiff3",
+    "finish-args": [
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--socket=system-bus",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
does it really need "host" access?

Reply:
We should try with nothing here as the portals should be enough.